build.gradle

```diff
@@ -19,19 +22,37 @@ repositories {
 dependencies {
     implementation libs.hec.monolith
     implementation libs.hec.nucleus.metadata
-    implementation 'com.itextpdf:itextpdf:5.5.13.2'
-
+    implementation 'org.swinglabs:pdf-renderer:1.0.5'
+    implementation 'dom4j:dom4j:1.6.1'
+    implementation 'crimson:crimson:1.1.3'
+    // implementation 'org.jfree:jfreechart:1.5.4'
     implementation files('dep-local/jfreechart.jar')
-    implementation files('dep-local/pdf-renderer.jar')
     implementation files('dep-local/dsm2-input-model.jar')
-    implementation files('dep-local/crimson.jar')
-    implementation files('dep-local/dom4j-1.6.1.jar')
     implementation files('dep-local/iText.jar')
     implementation files('dep-local/itext-hyph-xml.jar')
+    runtimeOnly 'mil.army.usace.hec:javaHeclib:7-IS-2-win-x86_64@zip'
+}
+
+task extractDll(type: Copy) {
+    def zipFile = configurations.runtimeClasspath.find { it.name.endsWith('.zip') }
+    from zipTree(zipFile)
+    into "$buildDir/dll"
 }
 
-jar {
-    manifest {
-        attributes 'Main-Class': 'gov.ca.dwr.callite.Main'
+startScripts {
+    doLast {
+        unixScript.text = unixScript.text.replace('DEFAULT_JVM_OPTS=""', 'DEFAULT_JVM_OPTS="-Djava.library.path=$projectDir/dll"')
```

Comment:
you're building a unix dist but not pulling unix HecLib DLL?

Reply:
You have your `distTar.dependsOn`, I think if you just skip that you don't have to set the `unixScript.text`

```
tasks.named('distTar').configure { enabled = false }
```